CMakeLists.txt

```diff
@@ -34,6 +34,9 @@ set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 
 if (APPLE)
     add_compile_options(-Wno-deprecated-declarations -Wrange-loop-analysis)
+endif()
+
+if (CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
So why dont I or CircleCI have any errors related to this ? Isnt this related to your local env. ?

Reply:
why would you fix something related to your local setup in repo ?

Reply:
this is issue of clang, it does not break anything and it can fix future issues